app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | Exception:

```

Comment:
According to the checklist, it's recommended to use type annotations. Here, you should specify the type of the dictionary `visitor`. It will help to understand what keys and values it can have. For example, you can use `typing.Dict` or create a `dataclass` or `NamedTuple` for the visitor.